### ISSUE TITLE
`Stop request, waiting for next inference & saving to occur...` but then no inf. could occur due to user error

### ISSUE THREAD

**Issue by MMathisLab** [AdaptiveMotorControlLab/CellSeg3d#25]
Hey @C-Achard one minor thing is if someone makes an error, like loads "labels" instead of images for inference, and then hits run, of course there is an error, but then I cannot stop the GUI; I have to crash it out to restart. The "Stop" should be a HARD stop overwrite, not waiting until "next epoch," as in this case one hits an infinite wait time :D. #MackenzieProofingTheGUI 

We also might want a more intuitive error -- ie, "sorry I can't handle" is great, but maybe add "be sure to check your input data is correct!"

```Number of threads has been set to 1 for macOS
RuntimeWarning: RuntimeError in aborted thread: /Users/svc-dashboard/D/P/ITKPythonPackage/ITK-source/ITK/Modules/IO/TIFF/src/itkTIFFImageIO.cxx:503:
ITK ERROR: TIFFImageIO(0x7fd11b0a8490): Unable to read tiff file: Sorry, can not handle images with 64-bit samples!
/Users/mwmathis/Downloads/for testing/dataset/testtest
Stop request, waiting for next inference & saving to occur...
Stop request, waiting for next inference & saving to occur...
```

**Comment by C-Achard** [comment]
Sure, I can work on a "hard reset", that has been an issue for a while. I'll work on it once I'm done with the inference on a single layer feature request.

**Comment by C-Achard** [comment]
Fixed in #26
